File: Cwn.Doe.BusinessModels/Entities/EmployerWorkplace.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Cwn.Doe.BusinessModels.Entities
{
    public class EmployerWorkplace
    {
        public virtual int EWID { get; protected set; }
        public virtual int Order { get; protected set; }
        public virtual string EMID { get; set; }
        public virtual string BUTypeCode { get; set; }

        /// <summary>
        /// District Code
        /// </summary>
        public virtual string EWLoc { get; set; }

        public virtual string EWName { get; set; }
        public virtual string EWHouse { get; set; }
        public virtual string EWMoo { get; set; }
        public virtual string EWBuilding { get; set; }
        public virtual string EWSoi { get; set; }
        public virtual string EWRoad { get; set; }
        public virtual string EWVillage { get; set; }
        public virtual string EWTamb { get; set; }
        public virtual string EWAmp { get; set; }
        public virtual string EWProv { get; set; }
        public virtual string EWPost { get; set; }
        public virtual string EWTel { get; set; }
        public virtual string EWFax { get; set; }
        public virtual string EWMobile { get; set; }
        public virtual int EWVersionNO { get; set; }

        public virtual void AssignOrder(int order)
        {
            Order = order;
        }

        public virtual void UpOrder()
        {
            --Order;
        }

        public virtual void DownOrder()
        {
           ++Order;
        }
    }
}

File: Web/app/model/EmployerWorkplace.js
Ext.define(AppConfig.appName + '.model.EmployerWorkplace', {
    extend: 'Ext.data.Model',
    xtype: 'employerworkplace',
    idProperty: 'EWID',
    fields: [
        { name: 'EWID' },
        { name: 'EMID', type: 'string' },
        { name: 'Order' },
        { name: 'EWHouse', type: 'string' },
        { name: 'EWBuilding', type: 'string' },
        { name: 'EWMoo', type: 'string' },
        { name: 'EWSoi', type: 'string' },
        { name: 'EWRoad', type: 'string' },
        { name: 'EWProvName', type: 'string' },
        { name: 'EWAmpName', type: 'string' },
        { name: 'EWTambName', type: 'string' },
        { name: 'EWPost', type: 'string' },
        { name: 'Address', type: 'string' }
    ]
});